addons/danger/functions/fnc_LeaderFlare.sqf

```diff
@@ -0,0 +1,95 @@
+#include "script_component.hpp"
```

Comment:
filename needs to start with a small L. have fun fixing that :P

Reply:
Oh lordy.